Makefile

```diff
@@ -124,10 +135,13 @@ nightly-cli:
 ################################################################################
 # Code generation: To be run after modifications to API types                  #
 ################################################################################
+APIMACHINERY_PKGS=k8s.io/apimachinery/pkg/util/intstr,+k8s.io/apimachinery/pkg/api/resource,+k8s.io/apimachinery/pkg/runtime/schema,+k8s.io/apimachinery/pkg/runtime,k8s.io/apimachinery/pkg/apis/meta/v1,k8s.io/api/core/v1,k8s.io/api/batch/v1
 
 .PHONY: codegen
-codegen:
-	buf generate api
+codegen: deps-tools codegen-proto codegen-controller codegen-ui codegen-docs
```

Comment:
The `hack-codgen` target runs `make codegen` inside a container that already has all the tools pre-installed.

You should probably update `hack-codegen` to run `make codegen-proto codegen-controller codegen-ui codegen-docs` instead.